notebooks/features/hyperparameter_tuning/HyperOpt-SynapseML.ipynb

```diff
@@ -100,14 +127,13 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from synapse.ml.core.platform import *\n",
+    "# Set MLflow experiment.\n",
     "\n",
-    "if running_on_synapse_internal():\n",
-    "    experiment_name = \"hyperopt-synapseml\"\n",
-    "else:\n",
-    "    experiment_name = \"/Shared/hyperopt-synapseml\"\n",
+    "if running_on_synapse():\n",
+    "    from notebookutils.mssparkutils import azureML\n",
     "\n",
-    "# Set MLflow experiment.\n",
+    "    ws = azureML.getWorkspace(\"AzureMLService1\")\n",
```

Comment:
Likewise here could you pull this into the top cell called prerequisites where you show people how to set this up on synapse